odoo/addons/test_lint/tests/test_l10n.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import ast
+import itertools
+import os
+
+from . import lint_case
+
+
+class L10nChecker(lint_case.NodeVisitor):
+    def matches_tagged(self, node):
+        if isinstance(node, ast.Call):
+            if isinstance(node.func, ast.Attribute):
+                return node.func.attr == 'tagged'
+            if isinstance(node.func, ast.Name):
+                return node.func.id == 'tagged'
+        return False
+
+    def visit_ClassDef(self, node):
+        tags = set()
+        for deco in node.decorator_list:
+            if self.matches_tagged(deco):
+                tags |= {arg.value for arg in deco.args}
+        if (
+            (len({'post_install_l10n', 'external_l10n'} & tags) != 1)
+            or ('post_install_l10n' in tags and 'post_install' not in tags)
+            # or ('post_install_l10n' not in tags and 'post_install' in tags)
```

Comment:
Why not this one ?

Reply:
Can't `external` tags be either `at_install` either `post_install`?

It would be great to have a mechanism that can take multiple specs
For instance:
* `post_install+l10n` instead of `post_install_l10n`
* `external+l10n` instead of `external_l10n`
* `external+at_install`

It would prevent having `at_install_external_l10n`, `at_install_l10n`, `post_install_external_l10n`, `post_install_l10n`,... 

This linter could also check that we are removing `standard` when we are adding `external`; same thing for `at_install`/`post_install`

Reply:
It stays a bit weird that this is a job for the linter, but if not other option...

Reply:
@jco-odoo  I think it's a perfect job for the linter as it checks for suspicious things in code.

@william-andre I agree with the last part that the linter could check removal of `standard` ...  but then you should rename it to a more generic name like `TestTagLinter` ...

For the rest, I'm don't agree, it seems that you just want shorter tags.

Reply:
> For the rest, I'm don't agree, it seems that you just want shorter tags.

And a number of tags not growing exponentially